connectors/spark/src/test/java/io/unitycatalog/connectors/spark/FakeFileSystem.java

```diff
@@ -0,0 +1,87 @@
+package io.unitycatalog.connectors.spark;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.*;
+import org.apache.hadoop.util.Progressable;
+
+import java.io.IOException;
+
+public class FakeFileSystem extends RawLocalFileSystem {
```

Comment:
can you add class docs to explain the purpose of this class? 
Also, maybe rename this to "CredentialTestFileSystem" to make its purpose more intuitive than just "Fake"?